lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -15,35 +15,14 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 
           <footer class="list-group-item-footer">
 
-            <span class="label label-warning"
-                ng-show="modal.constraints.isDriveLargeEnough(drive, modal.state.getImage()) && !modal.constraints.isDriveLocked(drive) && !modal.constraints.isDriveSizeRecommended(drive, modal.state.getImage())">
-                <i class="glyphicon glyphicon-warning-sign"></i>
```

Comment:
Yeah, I thought I removed them a while back, but I noticed they were there. The icon removal reflects Zeplin.